data/pigui/views/mainmenu.lua

```diff
@@ -134,7 +134,19 @@ local function quitGame()
 end
 
 local function continueGame()
-	Game.LoadGame("_exit")
+	if Game.CanLoadGame('_exit') and ( Engine.GetAutosaveEnabled() or not Game.CanLoadGame('_quicksave') ) then
+		Game.LoadGame("_exit")
+	else
+		Game.LoadGame('_quicksave')
+	end
+end
+
+local function canContinue()
+	if Game.CanLoadGame('_exit') or Game.CanLoadGame('_quicksave') then
```

Comment:
While I'd prefer you use idomatic lua (see below), this style doesn't have any major problems and can be easily refactored later. Please do keep in mind for the future that there are much more elegant and simple ways to express a construct like this, though!
```lua
return Game.CanLoadGame('_exit') or Game.CanLoadGame('_quicksave')
```

Reply:
Thanks for the hint! Reminds me that I still need to learn lua properly :)